examples/qt_interface.py

```diff
@@ -14,14 +14,14 @@
 def read_lean_template(file_name: str = 'template.lean') -> Tuple[int, str]:
     """Read a template Lean file containing a single sorry.
     Returns the line number of the sorry and the content where
-    sorry is replaced by a string placeholder {}"""
+    sorry is replaced by a string placeholder REPLACE_ME"""
```

Comment:
This docstring should be updated.

Reply:
OK, I changed this into 
    """Read a template Lean file containing at least one sorry.
    Returns the line number of the first sorry"""